media/base/demuxer.h

```diff
@@ -34,6 +34,9 @@ enum class DemuxerType {
   kFrameInjectingDemuxer,
   kStreamProviderDemuxer,
   kManifestDemuxer,
+#if defined(STARBOARD)
+  kProgressiveDemuxer,
```

Comment:
Consider using `kFFmpegDemuxer` directly, and add comment where it is used mentioning that it's used in Chromium for progressive demuxer.